### DISCUSSION TITLE
Add Ability to Associate a Default Grinder to a Brewing Method

### DISCUSSION THREAD

**Discussion by Corb3t** [graphefruit/Beanconqueror#524] [Ideas]
It would be nice if I could associate a default grinder to specific brewing methods. I have 2 grinders, and typically use the same grinder for espresso vs pourover, so being able to associate a default grinder would be nice.

**Comment by graphefruit** [comment]
Hey @Corb3t,
this is possible, but with an other style:
You can define Default-Parameters.
This means, when you add a new brew, it will set the last used values.
You can also customize the preparation method, to say which default-parameters you want to use.
So you would just need to do one shot espresso, set your grinder, and one for filter and set your grinder.
And customize these two preparation methods and set grinder as default set.

Also you can directly add a new espresso or pourover when you press long on "+" it will open a popup.
With the new 6.4 version there will be a new button for this.

![image](https://user-images.githubusercontent.com/6260244/233035431-17abee18-d0af-4712-95d4-d6ff4b8ff6cc.png)
![image](https://user-images.githubusercontent.com/6260244/233035589-3dda2c0f-fa18-41bb-ad1d-5c2a383ecf86.png)

Let me know if this helps you!

Best
Lars

**Reply by Corb3t** [reply]
Hi @graphefruit,

This is a great feature, thank you so much! I am concerned about two pre-filled fields when I use this function:

1) the default "bean type" isn't selecting the bean I previously used, it appears to always select a different bean

![image](https://user-images.githubusercontent.com/43692728/234312493-3870c8f8-afa9-4403-8692-8197a78e287e.jpeg)

![image](https://user-images.githubusercontent.com/43692728/234312541-1e72ce1e-bc30-4674-879e-c64a59dc635e.jpeg)
2) While brewing times should be auto cleared   

![image](https://user-images.githubusercontent.com/43692728/234312563-d8f395ce-0e5d-4cf9-b3d7-862b52368540.jpeg)

**Reply by graphefruit** [reply]
Did you customize your Preparation Method "Espresso"? 
In best case you could make an export and send me the Beanconqueror.json via e-mail: info@beanconqueror.com
And I have a look there.
